themes/default/content/blog/finops-with-pulumi/index.md

```diff
@@ -0,0 +1,126 @@
+Origin - the pipeline, process and/or system through which this came to be (eg ArgoCD, Jenkins, Pulumi)
+Business Unit - the team responsible, typically maps to the the cost center associated (eg Sales, Engineering)
+Environment - the teams/customers this deployment services (eg Dev, Staging, Production)
+Application Component - the part of the application that this resource serves (eg data pipeline, authentication, front end)
+
+You will now be able to report on and manage resources in those tag groups.  It’s advisable that you do not create too many required tags, or it will quickly get difficult to manage and it increases the likelihood of multiple tags applying, which reduces the insightfulness of the data.
+
+Read more about Tag Policy implementation with Pulumi below and for extended reading refer to this post, [Automatically Enforcing AWS Resource Tagging Policies](https://www.pulumi.com/blog/automatically-enforcing-aws-resource-tagging-policies/) from CEO, [Joe Duffy](https://www.pulumi.com/blog/author/joe-duffy/).
+
+## Pulumi Features in Context
+
+### Multi Language Components
+
+Multi-Language components allow you to create repeatable infrastructure patterns, modular abstractions of resources and/or groups of resources that meet your desired configuration standards.  Packaging up resources within and across clouds, implementing standard deployment best practices and making those components available to developers reduces the cognitive load that they have when building new applications from those pre-assembled, ready-to-run components.
+
+Use these guardrails to enforce Tag Policies, for example that every tag for “Production” is actually “Production” and not “Prod” or “production” (yes, case sensitivity matters).  Or you may ensure teams only deploy resources such as VMs to regions where you have allocated Reserved Instances or other financial benefits in your application architecture. With MLCs end users have access to your component and not to the underlying infrastructure resources, guaranteeing repeatabality, however, in order to realize the benefits of MLCs, your Developers must be empowered to leverage them, but limited from using other methods.
+
+## CrossGuard
+
+CrossGuard Policy as Code allows an Organization to define global rules for how resources may be provisioned and prevents resources from being provisioned by Pulumi in any other configuration.  Pulumi Stacks that do not meet the Organization’s financial, security, data protection or other pre-defined standards are prevented from provisioning any resources and the developer is instructed to correct their desired configuration or program in accordance with the Organization-wide policies.
+
+Policy as Code ensures that any Pulumi code written and run meets an Organization’s stated policies. It is the ultimate preventative weapon to prevent Financial, Security, or other placement and provisioning decisions that may lead to exposure or overrun.
```

Comment:
```suggestion
Policy as code ensures that any Pulumi code written and run meets an organization’s stated policies. It is the ultimate preventative weapon to prevent financial, security, or other placement and provisioning decisions that may lead to exposure or overrun.
```